reference/storage/file-signUrl.md

```diff
@@ -0,0 +1,60 @@
+Create a signed url for access to file
```

Comment:
typo - should this say "access to *a* file"

Reply:
done